templates/lune/lune.amp.html

```diff
@@ -17,8 +17,6 @@
 <html ⚡>
   {{> ../../utils/amp-page-head.snip.html}}
   <body class="lune">
-    {{> ../../components/navbar/navbar.snip.html}}
```

Comment:
The landing page is not including the navbar/sidebar anymore. It will be included in all other pages